src/main/scala/cromwell/engine/backend/jes/GcsAuth.scala

```diff
@@ -0,0 +1,71 @@
+package cromwell.engine.backend.jes
+
+import play.api.libs.json._
+
+object GcsAuth {
+  val boilerplateJson = Json.parse(s"""
+       |{
+       |  "auths" : {}
+       |}
+     """.stripMargin)
+
+  /**
+   * Add the JsObject parameters to the `auths` entry
+   */
+  def authTransformer(authJsons: Seq[JsObject]) = (__ \ 'auths).json.update {
+    __.read[JsObject] map { o =>
+      o ++ authJsons.reduce(_ ++ _)
+    }
+  }
```

Comment:
This is a bit specific to the play JSON library, which won't matter much when it'll be changed to use Spray. But in this library you have what they call "Transformers" which is a kind of abstract/generic val or method that you can (try to) apply to a json to perform operations on it.
This specific one would be in English : "Take the 'auths' object in the json you will be applied to, and update its content by adding all the `JsObject`s in the `authJsons` parameter".
So applied on the boilerplateJson
```
{
  "auths": {}
}
```
 with docker and gcloud objects as parameters you get 
```
{
  "auths": {
     "docker": {
         "account": ...,
         "token": ...
      },
      "gcloud": {
         "account": ...,
         "token": ...
      }
  }
}
```